package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "bat-client",
-  "version": "2.2.2",
+  "version": "2.2.3",
```

Comment:
before you do the "final commit":, please take a look at the master branch and make sure that you are bumping the version by one from whatever it is...

Reply:
Done